dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java

```diff
@@ -83,7 +84,7 @@
                                                                       new JavaLevelFilter()
     };
 
-    private static EE9PackageReplacementHelper ee9Helper;
+    private static EE9PackageReplacementHelper ee9Helper = new EE9PackageReplacementHelper();
```

Comment:
We can't init quite this early, it implies that in order to run any FAT (whether or not an EE9 repeat is used) this properties file will need to be present. Instead, lets just lazy-init the field, but do so using the no-args ctor